elements/lisk-transactions/src/schema.ts

```diff
@@ -100,14 +96,6 @@ export const baseTransaction = {
 			type: 'string',
 			format: 'publicKey',
 		},
-		signature: {
-			type: 'string',
-			format: 'signature',
-		},
-		signSignature: {
-			type: 'string',
-			format: 'signature',
-		},
 		signatures: {
 			type: 'array',
 			uniqueItems: true,
```

Comment:
Remove `uniqueItems` and `format` and checked in the code.
minItem is 1 and max is 64